packages/website/docs/RadioButtons.md

```diff
@@ -1,28 +1,53 @@
-## Guidelines
+# Best Practices
 
-(See also: **Guidelines for All Form Controls** - _coming soon!_)
+Use radio buttons to require user to select only one option from a short list.
 
-### General
+Radio buttons always come in set of 2 or more options.
 
--   The set of options should always have a title.
--   Use wording that indicates that the user can **only select one option**. Some users may not know the difference between radio buttons and checkboxes.
--   Put the most common options at the top of the list, unless your options have a strong logical ordering (such as prices or sizes).
--   Keep the number of options per set under 7.
+Do not use radio button for a simple binary choice (on/off), instead consider using a stand-alone checkbox.
+When choosing between only two options, make sure the options have explicitly different outcomes rather then opposing states.
 
-### Label
+-   E.g. Choosing between Landscape and Portrait layouts.
 
--   Try to limit label text to one short line.
--   Add a description or help hint only if necessary.
+Try to limit the number of available options under 7. If impossible, consider busing the [Single-Select](#/form/MultiSelect)
 
-### Behavior
+Options should be order by most relevant unless a more suited ordering rational applies (such as for sizes or security levels). If no rational particularly stands-out, use alphanumerical order.
 
--   Preselect the recommended option. If there is none, preselect the option most likely to be selected by users.
--   Never use radio buttons if the set of options is not exhaustive. The user **must** select an option.
+## Labeling
+
+Keep titles and labels short, preferably **under three words**.
+
+### Title
+
+A set of radio button always require a title to identify the type of options available.
+
+Use descriptive labels without action verb.
+
+-   E.g. Use "Print Layout" instead of "Select your print layout".
+
+### Labels
+
+Labels are used to name each option and should be self-explanatory as much as possible.
+
+Use a consistent writing style for all options in the list.
+
+
+## Feedback and validation
+
+Always preselect an option in the list as the default option. The default option can reflect the recommended path when we want to give assistance to the user, or the most commonly selected option if we want to expedite the tasks. If preselecting an option increases the risk of irrevertable changes or security issues (such as privileges), always use the least risky option as the default. 
```

Comment:
```suggestion
Always preselect an option as the default. The default option can identify:
- The recommended path when you want to assist the user
- The most commonly selected option when you want to help expedite the task.
If preselecting a default option increases the risk of irreversible changes or security issues, always use the least risky option as the default. 
```
@kpoulincoveo First sentence says "always preselect a default", but the last sentences could imply that preselecting a default is optional... I just want to make sure it's indeed mandatory to select a default option? (Don't commit this suggestion yet, I'll adjust the sentence depending on what you confirm 😉 )